near-sdk/src/collections/legacy_tree_map.rs

```diff
@@ -584,6 +584,36 @@ where
     }
 }
 
+#[cfg(feature = "expensive-debug")]
+impl<K> std::fmt::Debug for Node<K>
+where
+    K: std::fmt::Debug + Ord + Clone + BorshSerialize + BorshDeserialize,
+{
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("Node")
+            .field("id", &self.id)
+            .field("key", &self.key)
+            .field("lft", &self.lft)
+            .field("rgt", &self.rgt)
+            .field("ht", &self.ht)
+            .finish()
+    }
+}
+
+#[cfg(feature = "expensive-debug")]
+impl<K, V> std::fmt::Debug for LegacyTreeMap<K, V>
```

Comment:
Actually, we can just leave out the impl for `LegacyTreeMap`. It is deprecated and should probably be marked as such so we should not be making any functional changes to it